schema/shared/src/test/scala-2/zio/blocks/schema/DerivedOpticsSpec.scala

```diff
@@ -0,0 +1,369 @@
+package zio.blocks.schema
+
+import scala.language.reflectiveCalls
+import zio.test._
+
+object DerivedOpticsSpec extends ZIOSpecDefault {
+
+  // ===== Test Types =====
+
+  // Basic case class
```

Comment:
In order to force you to de-alias appropriately, you should do tests when a user passes a type alias to `DerivedOptics`, e.g. `type P = Person; ... object Person extends DerivedOptics[P]`, as well as any other cases where a user has forced a type alias.